symbols/overlay11.yml

```diff
@@ -390,6 +390,34 @@ overlay11:
         
         r0: [output] The entry
         r1: background ID
+    - name: GroundBg_Init
```

Comment:
There shouldn't be underscores in the name. That's allowed for known original function names, but not ones we just make up